packages/plugin-session-replay-browser/src/session-replay.ts

```diff
@@ -286,6 +315,7 @@ class SessionReplay implements SessionReplayEnrichmentPlugin {
     if (err) {
       this.config.loggerProvider.error(err);
     } else if (success) {
+      this.timeSinceLastSend = Date.now();
```

Comment:
time since implies it's the elapsed MS or S since last send - but this looks like it's MS since epoch instead... If i've understood this right, can we rename this?